Racecar_Lab/src/SolarCar.ts

```diff
@@ -0,0 +1,16 @@
+import { Racer } from "./Racer"
+
+export class SolarCar implements Racer {
```

Comment:
When a class implements an interface, you don't have to explicitly annotate types inside of the class. That's part of the convenience of interfaces. This applies to `GasCar` as well.